RELEASE-NOTES.md

```diff
@@ -4,8 +4,8 @@
 It also:
 * Refines and expands the user API;
 * Documents how to change the user configuration file in the Docker container;
-* Enhanced the blockchain state trees by using a kind of Merkle trees independent from the order of operations - i.e. Ethereum's "modified Merkle Patricia tree" modified by using SHA-256 (rather than Keccak) as hashing function;
-* Enhanced the persistence by using a database - i.e. Mnesia;
+* Enhances the blockchain state trees by using a kind of Merkle trees independent from the order of operations - i.e. Ethereum's "modified Merkle Patricia tree" modified by using SHA-256 (rather than Keccak) as hashing function;
```

Comment:
@uwiger  Spurious cherry-picked commit?

Reply:
Not sure. This is what I did:
```
uwpro-2:epoch uwiger$ git checkout PT-154763964-use-localhost-in-sname
Switched to branch 'PT-154763964-use-localhost-in-sname'
uwpro-2:epoch uwiger$ git fetch
remote: Counting objects: 10, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 10 (delta 3), reused 4 (delta 2), pack-reused 0
Unpacking objects: 100% (10/10), done.
From github.com:aeternity/epoch
 * [new branch]      PT-154763964_docker_hostname_docs -> origin/PT-154763964_docker_hostname_docs
   dd274ca..9107859  master     -> origin/master
uwpro-2:epoch uwiger$ git rebase origin/master
First, rewinding head to replay your work on top of it...
Applying: Use -sname epoch_xx@localhost + lager update
Using index info to reconstruct a base tree...
M	rebar.config
M	rebar.lock
Falling back to patching base and 3-way merge...
Auto-merging rebar.config
```

Reply:
Could you please try `git rebase -i origin/master` and remove the spurious commit and force-push?